tests/ips-state-1/test.yaml

```diff
@@ -28,12 +26,13 @@ checks:
       event_type: flow
       app_proto: http
       flow.alerted: false 
+      flow.action: pass
 
 - filter:
-    # We should see NO drops (or alerts) for http
+    # We should see NO drops for http
     count: 0
     match:
-      event_type: alert
+      event_type: drop
```

Comment:
Why not have 2 checks : 0 alerts, and 0 drops ?

Reply:
The one in lines 17-20 doesn't work?

Reply:
Oh yes, there was a duplicate check 👍